### DISCUSSION TITLE
transparent: add transparency

### DISCUSSION THREAD

**Discussion by MAFiA303** [Godiesc/firefox-gx#143] [Ideas]
i found this [repo](https://github.com/rkcpn/simple-transparent-firefox/blob/main/userChrome.css) it can add transparency to firefox.
thought you can ingest it into ffgx

**Comment by Godiesc** [comment]
Hi, AFAIK, transparency support on Windows OS ended a few versions ago, when Win7 support ended, I read that the ESR version of Firefox still has that feature; on MacOS I saw that it can still work with transparency, and on Linux I don't know.
